source/application/AMPIC887DriverServer/GCS2Commands/AMGCS2GetMinCommandablePositionCommand.cpp

```diff
@@ -51,56 +38,29 @@ bool AMGCS2GetMinCommandablePositionCommand::validateArguments()
 
 bool AMGCS2GetMinCommandablePositionCommand::runImplementation()
```

Comment:
Yes. I have been thinking about that a lot. There are certainly at least a few families of these commands, with the only difference in each family being the eventual call to the PI library. Will give this some though, because not maintaining all those command classes sounds nice to me.